index.bs

```diff
@@ -12626,14 +12629,17 @@ A {{DOMException}} is represented by a
 
 <h4 id="es-handling-exceptions">Handling exceptions</h4>
 
-None of the algorithms or processing requirements in the
-ECMAScript language binding catch ECMAScript exceptions.  Whenever
-an ECMAScript [=function object=] is invoked due
-to requirements in this section and that function call
-ends due to an exception being thrown, that exception
+Unless specified otherwise,
+whenever ECMAScript runtime semantics are invoked due
+to requirements in this document and
+end due to an exception being thrown, that exception
 must propagate to the caller, and if
 not caught there, to its caller, and so on.
 
+Per [[#conventions]], an algorithm specified in this document may intercept thrown exceptions, either by specifying
+the exact steps to take <dfn id="dfn-if-an-exception-was-thrown">if an exception was thrown</dfn>, or by explicitly
```

Comment:
If feels weird to integrate the conditional ("if") within the DFN. I'd be more confortable with:

```
<dfn id="dfn-an-exception-was-thrown">an exception was thrown</dfn>
```